_squirtle/mod.ts

```diff
@@ -70,29 +67,82 @@ const runLogin = async () => {
   await runUser(null, user, env)
 }
 
+const browse = (location: string): Promise<Deno.ProcessStatus> => {
+  return new Promise<Deno.ProcessStatus>((resolve, reject) => {
+    switch (process.platform) {
+      case 'aix':
+      case 'freebsd':
+      case 'linux':
+      case 'openbsd':
+        return Deno.run({
+          cmd: ['gp', 'preview', '--external', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .catch(() =>
+            Deno.run({
+              cmd: ['xdg-open', location],
+              stderr: 'null',
+              stdout: 'null',
+            })
+              .status()
+          )
+          .then(resolve)
+          .catch(reject)
+      case 'darwin':
+        return Deno.run({
+          cmd: ['open', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+      case 'win32':
+        return Deno.run({
+          cmd: ['cmd', '/c', 'start', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+    }
+  })
+    .then((status: Deno.ProcessStatus) => {
+      if (status.success)
+        console.log("opened %s", location)
```

Comment:
I don't think I want to log every time an external site opens. The site opens, that's the notification. Right?